locale/en.po

```diff
@@ -6755,6 +6752,9 @@ msgstr "Zoom in"
 msgid "ZOOM_OUT"
 msgstr "Zoom out"
 
+#~ msgid "CONTINUE_THRIVING"
```

Comment:
Why is this text deleted? I thought that this was pretty clever way to start the multicellular stage.